packages/ocplib-endian/ocplib-endian.1.2/opam

```diff
@@ -17,6 +17,7 @@ depends: [
   "ocaml" {>= "4.03.0"}
   "cppo" {>= "1.1.0" & build}
   "dune" {>= "1.0"}
+  "base-bytes"
```

Comment:
base-bytes is there already. Is that something to do with dune?

Reply:
The fix in the past was a cumbersome set of bounds (see https://github.com/ocaml/opam-repository/wiki/PR-checks#some-common-failures) but here it seems that it should have not been a problem